src/crimson/os/seastore/segment_cleaner.cc

```diff
@@ -685,23 +683,8 @@ SegmentCleaner::_retrieve_backref_extents(
     Cache::backref_extent_buf_entry_t::cmp_t> &&backref_extents,
   std::vector<CachedExtentRef> &extents)
 {
```

Comment:
_retrieve_backref_extents() can be removed

Reply:
yeah